extensions/index.adoc

```diff
@@ -0,0 +1,48 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="extensions-overview"]
+= Extensions overview
+include::_attributes/common-attributes.adoc[]
+:context: olmv1-about
+
+toc::[]
+
+:FeatureName: {olmv1}
+include::snippets/technology-preview.adoc[]
+
+Extensions enable cluster administrators to extend capabilities for users on their {product-title} cluster.
+
+{olm-first} has been included with {product-title} 4 since its initial release. {product-title} 4.17 includes components for a next-generation iteration of {olm} as a Generally Available (GA) feature, known during this phase as _{olmv1}_. This updated framework evolves many of the concepts that have been part of previous versions of {olm} and adds new capabilities.
+
+[id="olmv1-highlights_{context}"]
+== Highlights
+
+Administrators can explore the following highlights:
+
+Fully declarative model that supports GitOps workflows::
+{olmv1} simplifies extension management through two key APIs:
++
+--
+* A new `ClusterExtension` API streamlines management of installed extensions, which includes Operators via the `registry+v1` bundle format, by consolidating user-facing APIs into a single object. This API, provided as `clusterextension.olm.operatorframework.io` by the new Operator Controller component, empowers administrators and SREs to automate processes and define desired states by using GitOps principles.
++
+[NOTE]
+====
+Earlier Technology Preview phases of {olmv1} introduced a new `Operator` API; this API is renamed `ClusterExtension` in {product-title} 4.16 to address the following:
```

Comment:
```suggestion
Earlier Technology Preview phases of {olmv1} introduced a new `Operator` API; this API is renamed `ClusterExtension` in {product-title} 4.16 to address the following improvements:
```
Always try to add a noun after "the following", unless you're referring to "the following" in the sense of "I have a huge following on Twitter".